lib/stronger_parameters/constraints/file_constraint.rb

```diff
@@ -0,0 +1,13 @@
+require 'stronger_parameters/constraints'
+
+module StrongerParameters
+  class FileConstraint < Constraint
+    def value(v)
+      if [File, StringIO].include? v.class
```

Comment:
and ActionDispatch::Http::UploadedFile

https://github.com/rails/rails/blob/4e8880d29058ceb0549031c3f6fb95a88a26712e/actionpack/lib/action_dispatch/http/upload.rb#L68